packages/teams-js/src/internal/communication.ts

```diff
@@ -706,6 +711,19 @@ function handleParentMessage(evt: DOMMessageEvent): void {
     if (callbackId) {
       const callback = CommunicationPrivate.callbacks.get(callbackId);
       logger('Received a response from parent for message %i', callbackId);
+
+      // Send performance metrics information of message delay
```

Comment:
nit: not sure if there is any guidance on how the code is structured in teamsjs.. personally it feels this code can be extracted in a telemetry helper function to keep the communication.ts file less lengthy.

Reply:
I agree with this, I was also worried about clustering a lot the communication layer with small stuff related to telemetry. Will add a utility function instead thanks!